src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/SecurityController.php

```diff
@@ -0,0 +1,307 @@
+     * )
+     *
+     * @param int $sessionId
+     *
+     * @return RedirectResponse
+     */
+    public function deleteCustomerSessionAction(int $sessionId): RedirectResponse
+    {
+        try {
+            $deleteSessionCommand = new DeleteCustomerSessionCommand($sessionId);
+
+            $this->getCommandBus()->handle($deleteSessionCommand);
+
+            $this->addFlash('success', $this->trans('Successful deletion', 'Admin.Notifications.Success'));
+        } catch (CoreException $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+        }
+
+        return $this->redirectToRoute('admin_security_sessions_customer');
+    }
+
+    /**
+     * Bulk delete customer session.
+     *
+     * @AdminSecurity(
+     *     "is_granted('delete', request.get('_legacy_controller')~'_')",
+     *     message="You do not have permission to edit this."
+     * )
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function bulkDeleteCustomerSessionAction(Request $request): RedirectResponse
+    {
+        $sessionIds = $request->request->get('security_session_customer_bulk');
+
+        try {
+            $deleteSessionsCommand = new BulkDeleteCustomerSessionsCommand($sessionIds);
+
+            $this->getCommandBus()->handle($deleteSessionsCommand);
+
+            $this->addFlash('success', $this->trans('Successful deletion', 'Admin.Notifications.Success'));
+        } catch (CoreException $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+        }
+
+        return $this->redirectToRoute('admin_security_sessions_customer');
+    }
+
+    /**
+     * Bulk delete employee session.
+     *
+     * @AdminSecurity(
+     *     "is_granted('delete', request.get('_legacy_controller')~'_')",
+     *     message="You do not have permission to edit this."
+     * )
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function bulkDeleteEmployeeSessionAction(Request $request): RedirectResponse
+    {
+        $sessionIds = $request->request->get('security_session_employee_bulk');
+
+        try {
+            $deleteSessionsCommand = new BulkDeleteEmployeeSessionsCommand($sessionIds);
+
+            $this->getCommandBus()->handle($deleteSessionsCommand);
+
+            $this->addFlash('success', $this->trans('Successful deletion', 'Admin.Notifications.Success'));
+        } catch (CoreException $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+        }
+
+        return $this->redirectToRoute('admin_security_sessions_employee');
+    }
+
+    /**
+     * Get human readable error for exception.
+     *
+     * @return array
+     */
+    protected function getErrorMessages(): array
+    {
+        return [
+            SessionNotFoundException::class => $this->trans(
```

Comment:
You should handle the other exceptions as well like `CannotBulkDeleteCustomerSessionException` or other failures exception
I think some generic error messages exist with placeholders for failed bulked IDs